hello_world_on_chain/test/user.ts

```diff
@@ -69,11 +69,10 @@ describe("Local Test", function () {
         const publicKey = evmLinkData.publicKey;
 
         const {userContract} = await loadFixture(deployUser);
-        await userContract.setTASigningKeyAddress(publicKey as any);
 
         // when
         const ta = evmLinkData.transitiveAttestation;
-        const tx = await userContract.verifyAttestedFnCallClaims(ta as any)
+        const tx = await userContract.demo(publicKey, ta)
```

Comment:
Updated the name of the function that the user calls.  I thought it made it a little more clear what is going on, alternative names welcome.